setup.py

```diff
@@ -288,8 +288,15 @@ def run(self):
     'numpydoc',
 ]
 
+extras_require = {
+    'test:python_version == "2.7"': ['mock'],
+    'test': ['pytest', 'pytest-qt', 'pytest-cov', 'mock', 'coveralls',
+             'ciocheck', 'flaky', 'pandas', 'scipy', 'sympy', 'pillow'],
```

Comment:
But Travis is complaining about the lack of a package for ciocheck.

In any case, pytest doesn't run ciocheck, so it's not really a test dependency.